### PR TITLE
Constant base value

### LINKED ISSUE #3923: Change base entry constant storage cost to be 0.01 massa

### PR DESCRIPTION
**Pull request by AurelienFT**
* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

Fix #3923

### CODE REVIEW

File: massa-protocol-exports/Cargo.toml [hunk 1]
```diff
@@ -10,7 +10,7 @@ thiserror = "1.0"
 nom = "7.1"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
-peernet = { git = "https://github.com/massalabs/PeerNet", rev = "0ad0e2e" }
+peernet = { git = "https://github.com/massalabs/PeerNet", branch = "debuig" }
```
Comment (modship) [inline massa-protocol-exports/Cargo.toml:13]: maybe provide commit revision instead of branch ?

File: massa-node/Cargo.toml [hunk 2]
```diff
@@ -17,7 +17,7 @@ tracing = { version = "0.1", features = [
     "max_level_debug",
     "release_max_level_debug",
 ] }
-peernet = { git = "https://github.com/massalabs/PeerNet", rev = "0ad0e2e" }
```
Comment (modship) [inline massa-node/Cargo.toml]: same comment

**Review by modship — APPROVED** [review]

File: massa-protocol-exports/Cargo.toml (hunk 1 above)
Comment (AurelienFT) [reply massa-protocol-exports/Cargo.toml]: Done

File: massa-node/Cargo.toml (hunk 2 above)
Comment (AurelienFT) [reply massa-node/Cargo.toml]: Done

**Pull request merged**